packages/cap-config-writer/src/cap-writer/package-json.ts

```diff
@@ -53,12 +52,12 @@ async function updateScripts(
     packageJsonPath: string,
     projectName: string,
     appId: string,
-    enableNPMWorkspaces?: boolean
+    enableNPMWorkspaces?: boolean,
+    isCdsUi5PluginEnabled?: boolean
```

Comment:
AFAIK one of the criteria for cds ui5 plugin is workspaces being enabled. So we dont need both options?